README.md

```diff
@@ -116,6 +112,13 @@ $ ayb client profile marcua
  Adam Marcus  |             |              |          | http://marcua.net 
 ```
 
+Note that the command line also saved a configuration file (on Linux,
+to a place like `$HOME/.config/ayb/ayb.json`) for your convenience so
```

Comment:
Maybe adding three bullet points for Linux, MacOS and Windows would be useful for some people

Reply:
Done!